validation-test/stdlib/POSIXErrorCodeLinux.swift

```diff
@@ -0,0 +1,53 @@
+// RUN: %target-run-simple-swift
+// REQUIRES: executable_test
+//
+// UNSUPPORTED: OS=macosx
+// UNSUPPORTED: OS=ios
+// UNSUPPORTED: OS=tvos
+// UNSUPPORTED: OS=watchos
+// UNSUPPORTED: OS=linux-androideabi
+
+// REQUIRES: OS=linux-gnu
```

Comment:
Sure. It will just take a while for all the error codes. There are more on Linux than on OS X. I think only the first 35 are actually POSIX.